Src/Web/Web.Shared.Net/Implementation/WebEventSource.cs

```diff
@@ -373,7 +374,7 @@ public void InitializeHasNotBeenCalledOnModuleYetVerbose(string appDomainName =
                 38,
                 this.ApplicationName);
         }
-
```

Comment:
no change in this file right?

Reply:
I'll revert these changes since they're not needed anymore